src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -1,15 +1,29 @@
 {
   "version": "@WORKLOAD_VERSION@",
   "workloads": {
-    "microsoft-android-sdk-full": {
-      "description": "Android SDK",
+    "android": {
+      "description": "Microsoft Android SDK for .NET",
       "packs": [
         "Microsoft.Android.Sdk",
-        "Microsoft.Android.Sdk.BundleTool",
         "Microsoft.Android.Ref",
+        "Microsoft.Android.Runtime.android-arm",
+        "Microsoft.Android.Runtime.android-arm64",
+        "Microsoft.Android.Runtime.android-x86",
+        "Microsoft.Android.Runtime.android-x64",
         "Microsoft.Android.Templates"
       ],
       "extends" : [ "microsoft-net-runtime-android" ]
+    },
+    "android-full": {
+      "description": "Microsoft Android SDK for .NET with AOT support",
+      "packs": [
+        "Microsoft.Android.Sdk.BundleTool"
```

Comment:
@jonathanpeppers so my guess is that bundle tool will not be installed by default. But if a user wants to deploy to google store they will need it? Do they need to do anything manually to install bundle tool or will it be installed automatically when they try to use it? (I assume this is the google bundle tool thingy).

Reply:
The other thing is is this right to bundle this with aot? So it you want to deploy to google store, you need to download the entire AOT system as well regardless of it you use it or not?

Reply:
We could make an `android-bundletool` workload, so then if you had `<AndroidPackageFormat>aab</AndroidPackageFormat>` the error would say:

```
error NETSDK1147: To build this project, the following workloads must be installed: android-bundletool
error NETSDK1147: To install these workloads, run the following command: dotnet workload install android-bundletool
```

I split out this pack originally because `bundletool.jar` is 25 MB by itself:

https://github.com/google/bundletool/releases

Reply:
I'm just thinking of the most common workflow, which I would suspect is publishing to the google store. 
So given google is making that a requirement soon , I guess my question is should we be splitting out bundle tool at all? 

@jonpryor any thoughts on this one?

Reply:
Maybe we could go the other way with naming? Perhaps `android` could represent what is currently the `microsoft-android-sdk-full` workload, and we could add a new `android-minimal` that is the most lightweight option we can construct?

Reply:
I'll show these options to @jonpryor tomorrow, and hopefully we can get a decision on what to do.

I agree that if bundletool is *required* for Google Play soon, we should just put it in the box. Maybe it wouldn't even need to be its own pack?